src/main/kotlin/uk/gov/dluhc/printapi/database/repository/CertificateRepository.kt

```diff
@@ -16,10 +17,17 @@ interface CertificateRepository : JpaRepository<Certificate, UUID> {
 
     fun findByStatusAndPrintRequestsBatchId(certificateStatus: Status, batchId: String): List<Certificate>
 
-    fun findByStatus(certificateStatus: Status): List<Certificate>
+    fun findByStatusOrderByApplicationReceivedDateTimeAsc(certificateStatus: Status, pageable: Pageable): List<Certificate>
```

Comment:
The only way I could create a repository method that implements a limit with the `LIMIT` keyword in the SQL query is by passing in a `Pageable`. This means the caller (our service class) needs to know about and how to create a suitable `Pageable` which I'm not 100% happy about, but its the best I could do

Reply:
Unfortunately you cant put `LIMIT` in a JPQL query.
It (kinda) works if you make the query a native query, but then I think you need to make the method return a list of `Object` (or `Any` in kotlin 😁 ), and you have to mess around converting/casting/mapping (row mapper ?) the object yourself. It was messy and horrible

As mentioned, the only way I could get it working was with a `Pageable` which seems to be the way t'interwebs suggest you do it 😢